package.json

```
@@ -20,12 +20,19 @@
   },
   "homepage": "https://github.com/spring-guides/tut-react-and-spring-data-rest",
   "dependencies": {
+    "@material-ui/core": "^4.11.3",
+    "@material-ui/icons": "^4.11.2",
+    "@material-ui/lab": "latest",
+    "axios": "^0.21.1",
+    "bootstrap": "^4.6.0",
```

```suggestion
    "bootstrap": "^4.6.0",
```

Just a nit comment: seems like we are using Material Design for UI so we might not need to add a Bootstrap.